release/pipeline/dags/istio_common_dag.py

```diff
@@ -138,11 +181,14 @@ def MakeCommonDag(dag_args_func, name,
   )
 
   tasks = dict()
-  cmd_template = getBashSettingsTemplate(extra_param_lst)
+  init_gcb_env_prefix, copy_from_gcb_prefix = getBashSettingsTemplate(extra_param_lst)
 
   def addAirflowBashOperator(cmd_name, task_id, **kwargs):
     cmd_list = []
-    cmd_list.append(cmd_template)
+    if cmd_name == "get_git_commit_cmd":
```

Comment:
this looks hacky

can we just implement a stop to initialize gcb env or bake it in get_git_commit_cmd directly?

Reply:
simplified the code a bit using bootstrap files as discussed, and also added an init step